install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -191,16 +191,7 @@ istio-egressgateway:
     mountPath: /etc/istio/egressgateway-ca-certs
   #### Advanced options ########
   env:
-    # Set this to "external" if and only if you want the egress gateway to
-    # act as a transparent SNI gateway that routes mTLS/TLS traffic to
-    # external services defined using service entries, where the service
-    # entry has resolution set to DNS, has one or more endpoints with
-    # network field set to "external". By default its set to "" so that
-    # the egress gateway sees the same set of endpoints as the sidecars
-    # preserving backward compatibility
-    # ISTIO_META_REQUESTED_NETWORK_VIEW: ""
```

Comment:
isn't this a valid use case? what has changed? we even have a doc for this IIRC.

Reply:
Not found that, But from below, i think this is used for ServiceEntry endpoints resides in different network from what the gateway can access. But I have two question: 
1. in this case why do users set the endpoint within the network where gateway can not access?

2. Why `ISTIO_META_REQUESTED_NETWORK_VIEW` can be set only for gateway?

https://github.com/istio/istio/blob/3fbd356ecd2f065248ba6fc197c0490f7410049b/pilot/pkg/networking/core/v1alpha3/cluster.go#L427-L446